AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -2578,7 +2578,7 @@ e-marine.ru###subscribe_form_fixed
 kinokrad.co,kinoprofi.org,kinoprofi.vip###subscribe_sidebar
 ipress.ua###subscribe_us
 designmodo.com###subscribesinglepad
-cs-fundamentals.com###subscription-form
+novayagazeta.ru,cs-fundamentals.com###subscription-form
```

Comment:
`Ext-css` is better here. `novayagazeta.ru#?#footer > div[class] form[name="subscription-form"]:upward(1)`